proto/reaction.proto

```diff
@@ -916,6 +916,7 @@ message Data {
     string value = 1;
```

Comment:
Change this to `string_value`?

Reply:
Fixed by c177aa9; I chose not to also add an `int_value`, but we can do that to cover all of our bases